src/common/components/map/map-draw-control.tsx

```
@@ -0,0 +1,63 @@
+          metric: true,
+          showArea: false,
+        },
+      },
+      // edit: { featureGroup: new L.FeatureGroup() },
+    });
+
+    const drawnItems = new L.FeatureGroup();
+    map.addLayer(drawnItems);
+    map.addControl(drawControl);
+
+    const handleCreated = (e: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)